nncf/common/graph/operator_metatypes.py

```diff
@@ -22,11 +22,13 @@ class OperatorMetatype:
     :param name: The name of the operator.
     :param hw_config_names: The names of the hardware configurations.
     :param output_channel_axis: The axis along which the output channels of the operator are arranged.
+    :param ignored_input_ports: Input ports of the operations that should not be considered for purposes of compression.
     """
 
     name: str = ""
     hw_config_names: List[str] = []
     output_channel_axis: Optional[int] = None
+    ignored_input_ports: List[int] = []
```

Comment:
Shouldn't the required argument be placed before optional one?

Reply:
This is not a dataclass.